src/utils/config.ts

```diff
@@ -98,8 +122,10 @@ const errorOrConfig: t.Validation<IConfig> = IConfig.decode({
   )
     .map(_ => _.toLowerCase() === "true")
     .getOrElseL(() => false),
-  // -1 value fail IConfig decode if ENABLE_JWT is true
-  JWT_TOKEN_EXPIRATION: fromNullableE(-1)(process.env.JWT_TOKEN_EXPIRATION)
+  L1_TOKEN_EXPIRATION: fromNullableE(-1)(process.env.L1_TOKEN_EXPIRATION)
+    .chain(_ => IntegerFromString.decode(_).mapLeft(() => -1))
+    .fold(identity, identity),
+  L2_TOKEN_EXPIRATION: fromNullableE(-1)(process.env.L2_TOKEN_EXPIRATION)
```

Comment:
Why the default value is `-1`? If we have an `AA` this parameter is required and `NonNegativeInteger`, can we use an undefined default value?
```
  L1_TOKEN_EXPIRATION: IntegerFromString.decode(process.env.L1_TOKEN_EXPIRATION)
    .mapLeft(_ => undefined)
    .fold(identity, identity),
  L2_TOKEN_EXPIRATION: IntegerFromString.decode(process.env.L2_TOKEN_EXPIRATION)
    .mapLeft(_ => undefined)
    .fold(identity, identity)
```